Controls/ObjectExplorer/ObjectExplorerUC.xaml

```diff
@@ -100,7 +100,7 @@
                 </TreeView.ItemContainerStyle>
                 <TreeView.ItemTemplate>
                     <HierarchicalDataTemplate ItemsSource="{Binding Nodes, Converter={StaticResource sortConverter}}">
-                        <StackPanel Orientation="Horizontal" ToolTipService.ShowDuration="60000" Margin="2" AllowDrop="True" Drop="StackPanel_Drop" DragOver="StackPanel_DragOver">
+                        <StackPanel Orientation="Horizontal" ToolTipService.InitialShowDelay="500" ToolTipService.ShowDuration="60000" Margin="2" AllowDrop="True" Drop="StackPanel_Drop" DragOver="StackPanel_DragOver">
```

Comment:
I think we should remove ToolTipService properties from here, these should be set by the style in the containing application.

Reply:
I'm not sure if we can do that for the [ToolTipService](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.tooltipservice?view=netcore-3.1).

> You set the properties of the ToolTipService class by attaching them directly to the element that exposes the tooltip.

I guess I could try and do it by applying the style to the parent element and see what happens?

Reply:
Yes please, give it a try. If they're not inherited, see if you can set them in style in the containing application.